components/progress-bar/index.jsx

```diff
@@ -101,16 +106,26 @@ class ProgressBar extends React.Component {
 		const labels = assign({}, defaultProps.labels, this.props.labels);
 
 		return (
-			<div id={this.getId()}>
-				{this.getDescription({ labels })}
+			<div
+				id={this.getId()}
+				style={
+					this.props.orientation === 'vertical' ? { height: '200px' } : null
+				}
+			>
+				{this.props.orientation === 'horizontal' &&
+					this.getDescription({ labels })}
 				<div
```

Comment:
While we're here, do you mind adding the aria props `aria-valuemin="0" aria-valuemax="100" aria-valuenow={this.props.value} role="progressbar"`?  This should be straightforward, and then it would match the markup on the SLDS page.  Thanks!!